server.js

```diff
@@ -18,10 +37,48 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({
+    Message:"Happy Thoughts API!"
+   });
+});
+
+// To view all thoughts posted
+app.get("/thoughts", async (req, res) => {
+  const thoughts = await Thought.find()
+  .sort({createdAt: 'desc'})
+  .limit(20)
+  .exec();
+  res.json(thoughts);
+});
+
+// To post thoughts to the database
+app.post('/thoughts', async (req, res) => {
+  try {
+  const thought = new Thought({ 
+    message: req.body.message
+  }).save();
+  res.json(await thought);
+} catch (err) {
+  res.status (400).json({
+    message: 'Could not publish this thought',
+    errors: err.errors,
+  });
+ }
+});
+
+// To post new changes for the like-button
+app.post('/thoughts/:id/like', async (req, res) => {
+  try {
+    await Thought.updateOne({ _id: req.params.id }, { $inc: { hearts: 1 }});
+    res.status(201).json({ Success: 'Like has been added!' });
+  } catch (err) {
```

Comment:
Great job using updateOne and including { _id: req.params.id } in the await, 
Don't know how you avoided to use a const variable altogether but the count to the harts works!